### PR TITLE
New translations

### PR DESCRIPTION
**Pull request by gonzaloleon**
Add new strings to translate.

### CODE REVIEW

File: i18n/template.pot [hunk 1]
```diff
@@ -114,19 +128,19 @@ msgstr ""
 msgid "Add BitPay Account?"
 msgstr ""
 
-msgid "Add Me"
+msgid "Add Encryption?"
 msgstr ""
 
-msgid "Add Recipient"
+msgid "Add Enryption"
```
Comment (cmgustavo) [inline i18n/template.pot:134]: Please, review any typo and update the PR.

File: i18n/template.pot [hunk 2]
```diff
@@ -114,19 +128,19 @@ msgstr ""
 msgid "Add BitPay Account?"
 msgstr ""
 
-msgid "Add Me"
+msgid "Add Encryption?"
```
Comment (cmgustavo) [inline i18n/template.pot:131]: Is it different than "Add Encryption" without question mark?

**Review by cmgustavo — CHANGES_REQUESTED** [review]
Running `extract`, I got this: 

```
- src/pages/buy-crypto/crypto-offers/crypto-offers.ts
SKIP: Unknown argument type: 'TemplateExpression' NodeObject {
...
...
```

We should fix it before submit the POT file to Crowdin

File: i18n/template.pot (hunk 1 above)
Comment (gonzaloleon) [reply i18n/template.pot:134]: There is a typo in `src/components/encrypt-password/encrypt-password.html`, I'll fix it and include it in this PR.

File: src/pages/buy-crypto/crypto-offers/crypto-offers.ts [hunk 3]
```diff
@@ -526,9 +526,9 @@ export class CryptoOffersPage {
         if (err.errorCode) {
           switch (err.errorCode) {
             case 'validation.unsupportedCountry':
-              msg = this.translate.instant(
-                `Country not supported: ${this.selectedCountry.name}`
-              );
+              msg =
+                this.translate.instant('Country not supported: ') +
```
Comment (cmgustavo) [inline src/pages/buy-crypto/crypto-offers/crypto-offers.ts:530]: 👏

**Review by cmgustavo — APPROVED** [review]
ACK!

**Pull request merged**